lumicks/pylake/correlated_stack.py

```diff
@@ -151,7 +152,7 @@ def get_image(self, channel="rgb"):
 
         return np.stack([frame.data[slc] for frame in self], axis=0).squeeze()
 
-    def plot(self, frame=0, channel="rgb", show_title=True, **kwargs):
+    def plot(self, frame=0, channel="rgb", show_title=True, axes=None, **kwargs):
```

Comment:
Is the `axes` argument needed? Users can already override it using `plt.sca()`. At least I always felt like this was easier for third-party functions (ones that are not built into matplotlib). At least easier compared to always passing around the `axes` arguments.

For example, for builitin `pyplot` functions these two are equivalent:
```py
ax = ...

# class API
ax.plot(...)

# stateful API
plt.sca(ax)
plt.plot(...)
```

Reply:
This was something I put in for the confocal plotting (and now here for camera images) mostly for our benefit. Indeed you can use `plt.sca()` but with these compound plots and interactive plots/widgets we've been building it makes our code a lot simpler/cleaner if you can pass around the axes instance directly. For instance if you look [here](https://github.com/lumicks/pylake/pull/244#discussion_r781923759) I find the new syntax much cleaner

That being said, that's why I made it optional, so if the user doesn't supply it, it just gets `plt.gca()` so essentially nothing has changed from the user standpoint

Reply:
Ah, if you find the syntax more practical, don't mind my comment. I tend to prefer to minimize the args/API area to avoid extra testing. But if it makes the API more practical, go for it.